src/Raven.Studio/wwwroot/Content/css/modals.less

```diff
@@ -224,6 +224,12 @@
 }
 
 #licenseModal {
+    .licenseModalImg {
```

Comment:
why do we need this?

Reply:
well - we don't have to - I just thought the image looks nicer in the dialog when it is a bit **smaller**